src/jshint.js

```diff
@@ -2622,7 +2618,7 @@ var JSHINT = (function() {
       // first expression *or* the current group contains multiple expressions)
       if (!isNecessary && (first.left || ret.exprs)) {
         isNecessary =
-          (!isBeginOfExpr(preceeding) && first.lbp < preceeding.lbp) ||
+          (!isBeginOfExpr(preceeding) && first.lbp <= preceeding.lbp) ||
```

Comment:
do we want to do it this way, or does it make more sense to adjust the lbp marker of the arithmetic operators?